generator/views/glean_ping_view.py

```diff
@@ -294,6 +295,24 @@ def _make_dimension(
             ],
         }
 
+        if lookml["type"] == "time":
```

Comment:
Why can't we use the code from the generator here? https://github.com/mozilla/lookml-generator/blob/7a386b47097030f7876361e45ce374314053a93f/generator/views/lookml_utils.py#L42

Reply:
Actually that util code does generate dimensions for these datetime metric columns when `super().get_dimensions()` is called, but all metric dimensions from that are currently being excluded in favor of metric dimensions generated here.  One alternate approach would be to not exclude the upstream datetime metric dimensions and not generate datetime metric dimensions here, but there are differences between the two ways of generating dimensions which would be more trouble to work around (labels, hidden/not hidden, links). Overall I felt this was the least complicated way to get the datetime metric dimensions generated correctly that was still consistent with the other metric dimensions.